material/src/main/java/org/odk/collect/material/MaterialProgressDialogFragment.java

```diff
@@ -134,4 +171,12 @@ private void setArgument(String key, Serializable value) {
     public View getDialogView() {
         return dialogView;
     }
+
+    public boolean isShowing() {
+        AlertDialog dialog = (AlertDialog) getDialog();
+        if (dialog != null) {
+            return dialog.isShowing();
+        }
+        return false;
+    }
```

Comment:
Please convert this class to kotlin.

Reply:
I'm not sure why, but when i convert this class to kotlin I get following error when trying to build app:

> Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class org.odk.collect.android.formentry.RefreshFormListDialogFragment, unresolved supertypes: MaterialProgressDialogFragment
    class org.odk.collect.android.formentry.FormLoadingDialogFragment, unresolved supertypes: MaterialProgressDialogFragment
    class org.odk.collect.android.formentry.saving.SaveFormProgressDialogFragment, unresolved supertypes: MaterialProgressDialogFragment
    class org.odk.collect.android.gdrive.GoogleSheetsUploaderProgressDialog, unresolved supertypes: MaterialProgressDialogFragment

Reply:
Done.